docs/cli/kops_edit_cluster.md

```diff
@@ -7,7 +7,7 @@ Edit cluster.
 
 Edit a cluster configuration. 
 
-This command changes the cluster cloud specification in the registry. 
+This command changes the cluster cloud configuration in the registry. 
```

Comment:
Updated using "desired configuration" as proposed.
Regarding "in the registry", I think the word "registry" is a bit confusing, but I would still indicate where this is updated, I think it explains better what the command actually does.
"...in the S3 state config"?